pkg/apis/kubeless/v1beta1/http_trigger.go

```diff
@@ -32,11 +32,12 @@ type HTTPTrigger struct {
 
 // HTTPTriggerSpec contains func specification
 type HTTPTriggerSpec struct {
-	FunctionName  string `json:"function-name"` // Name of the associated function
-	HostName      string `json:"host-name"`
-	TLSAcme       bool   `json:"tls"`
-	RouteName     string `json:"route-name"`
-	EnableIngress bool   `json:"ingress-enabled"`
+	FunctionName    string `json:"function-name"` // Name of the associated function
+	HostName        string `json:"host-name"`
+	TLSAcme         bool   `json:"tls"`
+	RouteName       string `json:"route-name"`
+	EnableIngress   bool   `json:"ingress-enabled"`
+	IngressPathName string `json:"path-name"`
```

Comment:
I would call this just `Path`, it is explanatory enough

Reply:
but for me simply Path/path is also fine.

Reply:
yes, probably the whole object can be simplified, without thinking to much about it I would set it as:

```
type HTTPTriggerSpec struct {
	Function      string `json:"function"` // Name of the associated function
	Hostname    string `json:"hostname"`
	TLS               bool   `json:"tls"`
	Name            string `json:"name"`
	Path              string  `json:"path"`
}
```

The field `EnableIngress` is redundant since the existence of the CRD object means that it is an ingress object but I will leave it to @murali-reddy to have the final layout.

Reply:
+1 to @andresmgot's simplification above.

I think `Name` is too generic (I actually don't know what this is?), and I think a `tls` boolean is way too simplistic.  I suspect we want to change this to a TLSAcme bool, or have a way to pass annotations through, and/or specify the `tls` as `[]IngressTLS` as it is in the final `Ingress` object.